docs/src/reference/temporian/operators/window/moving_quantile.md

```diff
@@ -0,0 +1 @@
+::: temporian.EventSet.moving_quantile
```

Comment:
Also, can you create an entry in benchmark/benchmark_time.py to facilitate the benchmarking.

Reply:
*added it